src/service/impl/departures/index.ts

```diff
@@ -137,6 +141,52 @@ export default (
         return Result.err(new APIError(error));
       }
     },
+    async postStopQuayDepartures(
```

Comment:
I get that the HTTP needs to be POST, but this service implementation is it really `post`? No changes is stored and there are no side effects here. I think a more "correct" name would be something like `getStopQuayDeparturesByFavoriteLines` or something (yes long, but clearer)

Reply:
as `favorites` can be undefined here, is it meant that this service should be used both with and without favorites (so deprecating the other endpoint?).

Reply:
Yes. I guess one option here is to update the "old" getStopQuayDepartures instead of making a new function. It should act the same as the old one, and be backwards compatible, when favorites are not provided. Should I do that?

Reply:
I think that is good. Yes, it can introduce regression errors, but it is also better over time I think.

Reply:
Does this look OK? 1e808eb49a0d170ba36d4cf13e96655f4525377a

Reply:
Also tested this change to make sure that it works for older versions of the app.